src/renderer/features/governance/aggregates/locksPeriod.ts

```diff
@@ -0,0 +1,61 @@
+import { combine, createEvent, sample } from 'effector';
+import { createGate } from 'effector-react';
+
+import { type Chain } from '@/shared/core';
+import { nonNullable } from '@shared/lib/utils';
+import { networkModel } from '@entities/network';
+import { lockPeriodsModel } from '../model/lockPeriods';
+import { networkSelectorModel } from '../model/networkSelector';
+
+const lockPeriodsFetch = createGate<{ chain?: Chain }>();
```

Comment:
Are you sure about that name? I can't find a naming convention for it, but it seems like it should be a noun, not a verb